packages/fdkaac/fdkaac.0.3.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+homepage: "https://github.com/savonet/ocaml-fdkaac"
+bug-reports: "https://github.com/savonet/ocaml-fdkaac/issues"
+license: "GPL-2.0"
+dev-repo: "git+https://github.com/savonet/ocaml-fdkaac.git"
+build: [
+  ["./bootstrap"] {dev}
+  ["./configure" "--prefix=%{prefix}%"] {os != "macos"}
+  ["./configure" "OCAMLFLAGS=-cclib -L/usr/local/lib" "--prefix=%{prefix}%"]
+    {os = "macos"}
+  [make]
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "fdkaac"]
+depends: [
+  "ocaml"
+  "ocamlfind" {build}
+]
+conflicts: [
+  "liquidsoap" {< "1.4.0}
+]
+depexts: [
+  ["fdk-aac-dev"] {os-distribution = "alpine"}
+  ["libfdk-aac"] {os-distribution = "arch"}
+  ["fdk-aac-devel"] {os-distribution = "centos"}
+  ["fdk-aac-devel"] {os-distribution = "fedora"}
+  ["fdk-aac-devel"] {os-family = "suse"}
+  ["libfdk-aac-dev"] {os-distribution = "debian"}
+  ["libfdk-aac-dev"] {os-distribution = "ubuntu"}
+  ["fdk-aac"] {os = "macos" & os-distribution = "homebrew"}
+]
+synopsis: "Fraunhofer FDK AAC Codec Library"
+description: """
+The FDK AAC Codec Library For Android contains an encoder implementation of the
+Advanced Audio Coding (AAC) audio codec."""
```

Comment:
I think you need to add a newline between `.` and `"""`  as the CI complains with:
```
[ERROR] At /home/opam/.opam/repo/default/packages/fdkaac/fdkaac.0.3.0/opam:37:39:
        '.' is not a valid token
```

Reply:
Thanks, this is fixed now!